learntools/data_viz_easy/ex1.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+import matplotlib
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+from learntools.core import *
+
+file_path = '/Users/alexiscook/learntools/learntools/data_viz_easy/input/museum_visitors.csv'
+df = pd.read_csv(file_path, index_col="time", parse_dates=True)
+
+class LoadMuseumData(EqualityCheckProblem):
+    _var = 'museum_data'
+    _hint = "Use the `pd.read_csv()` function"
+    _solution = CS('museum_data = pd.read_csv(museum_file_path)')
+    _expected = df
+
+class ReviewData(EqualityCheckProblem):
+    _vars = ['ca_museum_jul18', 'avila_oct18']
+    _expected = [2620, 14658]
+    _hint = "Use the `tail()` command to print the last five rows. \
+    **After printing the last five rows**, \
+    The number of visitors in July 2018 for each museum can be found in the row marked `2018-07-01`. \
+    The number of visitors in October 2018 for each museum can be found in the row marked `2018-10-01`."
+    _solution = CS(
+"""# Print the last five rows of the data
+museum_data.tail()
+# How many visitors did the Chinese American Museum 
+# receive in July 2018? 
+ca_museum_jul18 = 2620
+# In October 2018, how many more visitors did Avila 
+# Adobe receive than the Firehouse Museum?
+avila_oct18 = 14658
+""")
+
+class PlotAll(CodingProblem):
+    _var = 'plt'
+    _solution = CS(
+"""# Line plot showing the number of visitors to each museum over time
+sns.lineplot(data=museum_data)
+""")
+    _hint = 'Use `sns.lineplot()`, and set `data=museum_data`.'
+
+    def check(self, passed_plt):
+        assert len(passed_plt.figure(1).axes) > 0, \
```

Comment:
This function has the expression `passed_plt.figure(1).axes[0]`  show up a few times.

It will be more readable if you put
```
main_axis = passed_plt.figure(1).axes[0]
legend_labels = main_axis.get_legend_handles_labels()
``` 
towards the top of the function.

Then expressions further down like `len(passed_plt.figure(1).axes[0].get_legend_handles_labels()[0]) == 4` becomes `len(legend_labels) == 4`
and 
` assert set(passed_plt.figure(1).axes[0].get_legend_handles_labels()[1]) == legend_set` becomes `assert set(legend_labels) == legend_set`

Reply:
sounds good!